src/main/java/org/prebid/server/privacy/model/Privacy.java

```diff
@@ -0,0 +1,39 @@
+package org.prebid.server.privacy.model;
+
+import lombok.Value;
+import org.apache.commons.lang3.ObjectUtils;
+
+@Value
+public final class Privacy {
+
+    private static final String DEFAULT_CONSENT_VALUE = "";
+    private static final String DEFAULT_GDPR_VALUE = "";
+    private static final String DEFAULT_CCPA_VALUE = "";
+
+    private static final Privacy EMPTY = Privacy.of(DEFAULT_GDPR_VALUE, DEFAULT_CONSENT_VALUE, DEFAULT_CCPA_VALUE);
+
+    private final String gdpr;
+
+    private final String consent;
+
+    private final String ccpa;
+
+    private Privacy(String gdpr, String consent, String ccpa) {
+        this.gdpr = gdpr;
+        this.consent = consent;
+        this.ccpa = ccpa;
+    }
+
+    public static Privacy of(String gdpr, String consent, String ccpa) {
+        final String validatedGdpr = ObjectUtils.notEqual(gdpr, "1") && ObjectUtils.notEqual(gdpr, "0")
```

Comment:
Kind of clean code stuff: i like the idea of `PrivacyExtractor`, but here we're adding some business logic to plain models since setting of `DEFAULT_GDPR_VALUE` is not the responsibility of a model. 
As well as another `DEFAULT_*` values. All those are related to HTTP query string params.

So, may be it make sense to extract this functionality to another place. What do you think?